ydb/core/blobstorage/base/blobstorage_vdiskid.cpp

```diff
@@ -24,7 +32,7 @@ namespace NKikimr {
     }
 
     bool TVDiskID::SameGroupAndGeneration(const NKikimrBlobStorage::TVDiskID &x) const {
-        return x.GetGroupID() == GroupID && x.GetGroupGeneration() == GroupGeneration;
+        return x.GetGroupID() == GroupID.GetRawId() && x.GetGroupGeneration() == GroupGeneration;
```

Comment:
it's better to use TGroupId::FromProto(&x, &NKikimrBlobStorage::TVDiskID::GetGroupID) == GroupID